server/constants/paths.ts

```diff
@@ -0,0 +1,23 @@
+const paths = {
+  START: '/',
+  SEARCH: '/orders',
+
+  INTEGRITY_ORDER: {
+    INDEX: '/integrity/orders',
```

Comment:
We could remove the `/orders` part of each path, shortening them to `/integrity/:legacySubjectId/subsection`.